wiki/osu!talk/en.md

```diff
@@ -1,95 +1,110 @@
-| 11 | [WubWoofWolf](https://osu.ppy.sh/users/39828) | Top Player | [Link](https://www.youtube.com/watch?v=XYzKlfvQt-w) |
-| 12 | [Ephemeral](https://osu.ppy.sh/users/102335) | Retired Staff Manager | [Link](https://www.youtube.com/watch?v=eXWmjo0-oyM) |
-| 13 | [ztrot](https://osu.ppy.sh/users/6347) | Media Creation | [Link](https://www.youtube.com/watch?v=8COmLt0IBRs) |
-| 14 | [Raiku](https://osu.ppy.sh/users/1525538) | Pro Player | [Link](https://www.youtube.com/watch?v=5P9FaFrS0CM) |
+| 9 | [Cyclone](https://osu.ppy.sh/users/18589) | Retired player/team member  | [Link](https://www.youtube.com/watch?v=jPUSY0FMw2E) |
+| 10 | [thelewa](https://osu.ppy.sh/users/475021) | Pro osu! player | [Link](https://www.youtube.com/watch?v=N7P-J-5LJzk) |
+| 11 | [WubWoofWolf](https://osu.ppy.sh/users/39828) | Pro osu! player | [Link](https://www.youtube.com/watch?v=XYzKlfvQt-w) |
+| 12 | [Ephemeral](https://osu.ppy.sh/users/102335) | Retired team manager | [Link](https://www.youtube.com/watch?v=eXWmjo0-oyM) |
+| 13 | [ztrot](https://osu.ppy.sh/users/6347) | Media creator | [Link](https://www.youtube.com/watch?v=8COmLt0IBRs) |
+| 14 | [Raiku](https://osu.ppy.sh/users/1525538) | Pro osu! player | [Link](https://www.youtube.com/watch?v=5P9FaFrS0CM) |
 | 15 | [smoogipooo](https://osu.ppy.sh/users/1040328) | Developer | [Link](https://www.youtube.com/watch?v=vG1yx1xVQsk) |
 | 16 | [OnosakiHito](https://osu.ppy.sh/users/290128) | QAT | [Link](https://www.youtube.com/watch?v=ZYby7r3YNPg) |
 | 17 | [RBRat3](https://osu.ppy.sh/users/307202) | osu!painter | [Link](https://www.youtube.com/watch?v=kSotXmkCN4I) |
 | 18 | [Andrea](https://osu.ppy.sh/users/33599) | Retired GMT/BN | [Link](https://www.youtube.com/watch?v=dKEOVBiljdc) |
-| 19 | [Doomsday](https://osu.ppy.sh/users/18983) | Pro Player | [Link](https://www.youtube.com/watch?v=0C74QeEcn_4) |
+| 19 | [Doomsday](https://osu.ppy.sh/users/18983) | Pro osu! player | [Link](https://www.youtube.com/watch?v=0C74QeEcn_4) |
 | 20 | [Tom94](https://osu.ppy.sh/users/1857058) | Developer | [Link](https://www.youtube.com/watch?v=ONnUrG4jrto) |
 | 21 | [Flanster](https://osu.ppy.sh/users/447818) | GMT | [Link](https://www.youtube.com/watch?v=nvGP5x9ZseM) |
 | 21.5 | [Blue Dragon](https://osu.ppy.sh/users/19048) | Mapper | [Link](https://puu.sh/cmOO3/a737a268da.mp3) |
-| 22 | [HappyStick](https://osu.ppy.sh/users/256802) | Pro Player | [Link](https://www.youtube.com/watch?v=zhAHOreuYp4) |
-| 23 | [Hayabusa](https://osu.ppy.sh/users/3104108) | Top UK Mania Player | [Link](https://www.youtube.com/watch?v=1C102Zzuyzg) |
-| 24 | [Kyonko Hiraza](https://osu.ppy.sh/users/444868) | Retired Pro Player | [Link](https://www.youtube.com/watch?v=6RhBqhhn9F0) |
-| 25 | [PortalLife](https://osu.ppy.sh/users/929134) | Player/Community Tournaments | [Link](https://www.youtube.com/watch?v=odGwuBwqcmc) |
-| 26 | [MillhioreF](https://osu.ppy.sh/users/941094) | Bagel Lord | [Link](https://www.youtube.com/watch?v=dO3kv8nutSI) |
```

Comment:
did these descriptions or the new ones come from anywhere? it feels like it was written this way on purpose

Reply:
i couldn't find from where were the professions taken, so i replaced them with actual meaningful descriptions (based on `People` articles and video descriptions)

probably the context is in the videos themselves, basing on this comment from the `Bagel Lord` episode:
![image](https://user-images.githubusercontent.com/19843905/102022870-ebc06300-3d89-11eb-866e-185056c80d01.png)